CHARTER.md

```diff
@@ -4,10 +4,12 @@ The CHERI SIG will work on a strategy for adding a capability based security mod
 
 Background:
 
-CHERI is a technology that extends instruction sets with new architectural features (so-called capabilities) to enable fine-grained memory protection and highly scalable software compartmentalization. CHERI was developed at the University of Cambridge and SRI and includes an instantiation for RISC-V (32- and 64-bit) using the custom opcode space.
+CHERI is a technology that extends instruction sets with new architectural features (so-called capabilities) to enable fine-grained memory protection and highly scalable software compartmentalization. CHERI was developed at the University of Cambridge and SRI and includes an instantiation for RISC-V (32- and 64-bit) using the custom opcode space. Microsoft has also announced an IoT-focused adaptation to be published in 2023.
 
 Scope:
 
- * Determine which which currently defined architectural CHERI-RISC-V features and instructions are required for a first standardized extensions
+ * Review currently defined architectural CHERI-RISC-V features and instructions from the SRI/Cambridge baseline specification to determine which are candidates for first standardized extensions
+ * Review changes made to RISC-V ISA baseline since CHERI-RISC-V was developed, to identify gaps where updating may be required -- e.g., around recent work on virtualization and vector extensions
+ * Develop a specification strategy that takes into account both the SRI/Cambridge baseline CHERI-RISC-V and Microsoft's microcontroller-focused variation, possibly involving a combined baseline specification alongside two specific encoding variations
```

Comment:
Again, maybe make this a bit more vague saying that we should consider other variations. I would be hesitant to name Microsoft's specification specifically if it has not yet been published.

Reply:
As we currently understand it, there will be two existing specifications on the table: The SRI/Cambridge specification (for 32/64-bit), and the derived Microsoft specification (for 32-bit small cores). The sets of tradeoffs in their implementations (and especially encodings) are quite different despite their both implementing the CHERI model. So I suspect in practice we’ll want to do what I’ve written above. We can not say that and just hand wave, but if the reason is that it’s not yet published, we just wait a few weeks until it is. And if it doesn’t turn up in a few weeks, then we can try again with more vague intentions. Or are there also other reasons not to specifically name the Microsoft spec that we should be considering?

Reply:
Ok, as long as there is a way to amend the charter in the way that you mention, then I am happy with keeping this in.